content/departments/product-engineering/process/new-user-experience.md

```diff
@@ -12,7 +12,7 @@ The Growth team will own the initial journeys of both new unauthenticated and au
 
 From time to time, the Growth team will need to alter these user journeys to accommodate advances in product functionality.
 
-The scope of Growth's efforts should not extend to the specifics of how to use individual product features, that would be up to the team that owns that feature.
+The scope of Growth's efforts should not extend to comprehensive training on individual product features, that would be up to the team that owns that feature.
```

Comment:
I like the word comprehensive. What about this, "While comprehensive training on individual product features is the domain and responsibility of the team that owns that feature, Growth is a critical stakeholder and participant in developing artifacts for new users as part of their initial journey adopting Sourcegraph."

Reply:
We should also probably remove Integrations from the header since the team is being split.

Reply:
Works for me, I've made that change.